programs/bpf_loader/src/syscalls.rs

```diff
@@ -1408,12 +1407,14 @@ where
     let mut accounts = Vec::with_capacity(account_keys.len());
     let mut refs = Vec::with_capacity(account_keys.len());
     for (i, ref account_key) in account_keys.iter().enumerate() {
-        let account =
-            invoke_context
-                .get_account(&account_key)
-                .ok_or(SyscallError::InstructionError(
-                    InstructionError::MissingAccount,
-                ))?;
+        let account = invoke_context.get_account(&account_key).ok_or_else(|| {
+            ic_msg!(
+                invoke_context,
+                "Instruction references an unknown account {:?}",
```

Comment:
Why use debug formatting over display formatting which would log the base58 address?